.changeset/nice-snakes-camp.md

```diff
@@ -0,0 +1,36 @@
+---
+"@salt-ds/core": minor
+---
+
+**_Theming and CSS updates_** with visual changes to Salt components:
+
+1. Minor changes in `Button` component width as font weight gets updated. Font weight for two button variants,`CTA` and `Primary` changed from bold to semibold. Expect components containing Salt `Button` to have similar change.
+
+```diff
+- --salt-actionable-cta-fontWeight
+- --salt-actionable-primary-fontWeight
+- --salt-actionable-secondary-fontWeight
++ --salt-text-action-fontWeight
+```
+
+![Button before and after](packages/core/stories/assets/buttonsOldAndNew.png)
```

Comment:
Is this the best place for the pictures? and should they be named using camel case?

Reply:
I have updated their names to use kebab case and moved them to a dedicated core/images folder, what do you think? @joshwooding